airflow/Dockerfile

```diff
@@ -10,33 +20,33 @@ COPY airflow/constraints-${PRODUCT}-python${PYTHON}.txt /tmp/constraints.txt
 # Requires implementation of https://github.com/apache/airflow/blob/2.2.5/scripts/docker/install_mysql.sh
 ENV AIRFLOW_EXTRAS=async,amazon,celery,cncf.kubernetes,docker,dask,elasticsearch,ftp,grpc,hashicorp,http,ldap,google,google_auth,microsoft.azure,odbc,pandas,postgres,redis,sendgrid,sftp,slack,ssh,statsd,virtualenv,trino
 
-RUN microdnf update \
-        && microdnf install -y \
-            gcc \
-            gcc-c++ \
-            python${PYTHON}-devel \
-            python${PYTHON}-pip \
-            python${PYTHON}-wheel \
-            openssl-devel \
-            cyrus-sasl-devel \
-            openldap-devel \
-            unixODBC-devel \
-            libpq-devel \
-        && microdnf clean all \
-        && python3 -m venv --system-site-packages /stackable/app \
-        && source /stackable/app/bin/activate \
-        && pip install --no-cache-dir --upgrade pip \
-        && pip install --no-cache-dir apache-airflow[${AIRFLOW_EXTRAS}]==${PRODUCT} --constraint /tmp/constraints.txt \
-        # Needed for pandas S3 integration to e.g. write and read csv and parquet files to/from S3
-        && pip install --no-cache-dir s3fs
-
-FROM prom/statsd-exporter:0.3.0@sha256:a9c27602d6f6b86527657922b6a87c12789f7f9b39a90f1513e8c665c941f26a as statsd-exporter
-FROM docker.stackable.tech/stackable/git-sync:v3.6.4 as gitsync-image
+RUN microdnf update && \
+    microdnf install \
+        cyrus-sasl-devel \
+        gcc \
+        gcc-c++ \
+        libpq-devel \
+        openldap-devel \
+        openssl-devel \
+        python${PYTHON}-devel \
+        python${PYTHON}-pip \
+        python${PYTHON}-wheel \
+        unixODBC-devel && \
+    microdnf clean all && \
+    python3 -m venv --system-site-packages /stackable/app && \
+    source /stackable/app/bin/activate && \
+    pip install --no-cache-dir --upgrade pip && \
+    pip install --no-cache-dir apache-airflow[${AIRFLOW_EXTRAS}]==${PRODUCT} --constraint /tmp/constraints.txt && \
+    # Needed for pandas S3 integration to e.g. write and read csv and parquet files to/from S3
+    pip install --no-cache-dir s3fs
```

Comment:
I would say we need s3fs in the final image, not the builder. Hupsi :innocent:

Reply:
Oh, sorry! I must have moved this accidentally. I'll move it back.

Reply:
I've looked. I think this is actually correct.
This is built/installed in a Virtualenv and we move the whole directory over to the final image later.